org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/id/IdCodec.java

```diff
@@ -235,4 +247,16 @@ protected Object[] parseComponents(Class<? extends IId> idClass, String[] rawCom
     }
     return components;
   }
+
+  /**
+   * Unregister type mapping between a string representation and the given {@code rawType}.
+   * <p>
+   * Note: The access to the type mapping data structure is not synchronized and therefore not thread safe. Use this
+   * method to set up the {@link IdCodec} instance directly after platform start and not to change the {@link IdCodec}
+   * behavior dynamically at runtime.
+   */
+  protected void unregisterRawTypeMapper(Class<?> rawType) {
```

Comment:
Position directly below the register method further above

Reply:
Done